Dockerfile

```diff
@@ -1,2 +1,23 @@
-FROM node:0.10-onbuild
-EXPOSE 3000
\ No newline at end of file
+FROM node:0.10
+
+RUN mkdir /usr/src/app
```

Comment:
An internal idiom we use (so no problem with you not being aware) is to install under `/opt/wftda/PROJECT`. I think that could work here as well?